tests/test_stats.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+import ompy as om
+import numpy as np
+import scipy.stats as stats
+from numpy.testing import assert_allclose
+
+
+def test_normal():
+    N = 10000
+    x = np.random.rand(N)
+
+    norm_native = np.array([om.stats.normal(xi) for xi in x])
```

Comment:
Don't you have a vectorized version now, too? So one could also test `normal_vec`. (and truncnormal (...))

Reply:
Added tests for the vectorised versions as well.